src/metaArray.cxx

```diff
@@ -1085,7 +1085,7 @@ MetaArray::M_ReadElements(std::ifstream * _fstream, void * _data, int _dataQuant
 bool
 MetaArray::M_WriteElements(std::ofstream * _fstream, const void * _data, std::streamoff _dataQuantity)
 {
-  bool            localData = false;
+  bool            localData;
```

Comment:
Shouldn't the variables themselves be removed then?

Reply:
No.  The are unconditionally set below with something like:

```c++
if(true){
localData = true;
} else {
localData=false;
}
```
The compiler identified that the initialization was never used.